### PR TITLE
Add aedat

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by isuruf — CHANGES_REQUESTED** [review]
Needs work on license. See https://github.com/conda-forge/conda-forge.github.io/issues/1052

**Comment by Tobias-Fischer** [comment]
Thanks for the pointer @isuruf.

Licenses packaged thanks to script found in https://github.com/conda-forge/staged-recipes/pull/13208

**Comment by Tobias-Fischer** [comment]
I now use the same `maturin` mechanism to build the `aedat` package as in https://github.com/conda-forge/pysyntect-feedstock, which enabled me to build the package on Windows as well. All licenses are contained as well.

It would be great if someone could have a look at the PR. Many thanks!

File: recipes/aedat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - maturin
+    - pip
```
Comment (chrisburr) [inline recipes/aedat/meta.yaml:21]: I'm not sure but are these needed in build?
```suggestion
```

**Pull request merged**